ert_shared/ensemble_evaluator/entity/ensemble_base.py

```diff
@@ -33,6 +33,12 @@ def get_active_reals(self):
     def get_metadata(self):
         return self._metadata
 
-    async def send_cloudevent(self, url, event):
-        async with websockets.connect(url) as websocket:
-            await websocket.send(to_json(event))
+    async def send_cloudevent(self, url, event, retry=0):
+        if retry == 10:
+            raise Exception(f"Could not send event {event}  to url {url}")
+        try:
+            async with websockets.connect(url) as websocket:
+                await websocket.send(to_json(event))
+        except Exception:
```

Comment:
Could this be written non-recursive? It may be a bit confusing reading the stacktrace for failures exhausting retries…